docs_user/modules/proc_adopting-compute-services-to-the-data-plane.adoc

```diff
@@ -281,14 +278,12 @@ ifeval::["{build}" == "downstream"]
     - redhat
 endif::[]
     - bootstrap
-    - download-cache
```

Comment:
please explain the removal?
In the architecture repo, this service presents in the most of the places alongside bootstrap.
@bshephar PTAL

Reply:
@pinikomarov according to https://issues.redhat.com/browse/OSPRH-9500?focusedId=26796751&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-26796751 and the issue's release notes, we should keep this service included. Please use the arch repo as the primary source of trueth